src/features/import/hooks/useGenderMapping.ts

```diff
@@ -3,7 +3,7 @@ import { useAppDispatch } from 'core/hooks';
 import { CellData, Column, ColumnKind } from '../utils/types';
 
 export const genders = ['f', 'm', 'o'] as const;
-export type Gender = typeof genders[keyof typeof genders];
```

Comment:
love that "length" was  a valid gender in the Zetkin Universe for some time, without anyone noticing 😂